operator/Makefile

```
@@ -397,13 +397,15 @@ ACTIVATE_PYTHON = python3 -m venv bundle_helpers/.venv ;\
 	pip3 install -r bundle_helpers/requirements.txt
 
 .PHONY: bundle
-bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
+bundle: yq manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
 # We hardcode the image reference to quay.io/stackrox-io/stackrox-operator. If this is overridden via
 # the IMG_REPO environment variable, the final reference will be injected in the bundle-post-process step.
 	$(OPERATOR_SDK) generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=quay.io/stackrox-io/stackrox-operator:0.0.1
 	cd config/scorecard-versioned && $(KUSTOMIZE) edit set image scorecard-test=quay.io/operator-framework/scorecard-test:$(OPERATOR_SDK_VERSION)
 	$(KUSTOMIZE) build config/manifests | $(OPERATOR_SDK) generate bundle $(BUNDLE_GEN_FLAGS)
+# Fix the createdAt annotation
+	YQ=$(YQ) $(YQ) -i '.metadata.annotations.createdAt = ""' bundle/manifests/rhacs-operator.clusterserviceversion.yaml
```

Nit: I think this bit is unnecessary?

```suggestion
	$(YQ) -i '.metadata.annotations.createdAt = ""' bundle/manifests/rhacs-operator.clusterserviceversion.yaml
```

FWIW, the `KUTTL=$(KUTTL)` is needed because we invoke kuttl from within kuttl tests.